slsDetectorSoftware/src/Module.cpp

```diff
@@ -1389,19 +1389,25 @@ void Module::setInjectChannel(const int offsetChannel,
     sendToDetector(F_SET_INJECT_CHANNEL, args, nullptr);
 }
 
-void Module::sendVetoPhoton(const int chipIndex, int *gainIndices,
-                            int *values) {
+void Module::sendVetoPhoton(const int chipIndex, std::vector<int> gainIndices,
```

Comment:
Can use a const& to the vector in order to avoid copying the data.

Reply:
done